### PR TITLE
update the old content of chinese website

### PR DESCRIPTION
**Pull request by yeya24**
fix a typo in line 173 Kubernets -> Kubernetes

### CODE REVIEW

**Comment by idealhack** [comment]
@yeya24 Thanks for your contribution! You are updating an outdated page, it looks like haven't been updated for one year and a half at least! Would you like to update it with the [English version](https://github.com/kubernetes/website/blob/master/content/en/docs/tasks/manage-gpus/scheduling-gpus.md)?

**Comment by yeya24** [comment]
Thanks for your reply @idealhack ! I missed that it is an outdated page. So should I close this pr by myself?

**Comment by idealhack** [comment]
@yeya24 No need to close this PR, what I mean is since you are fixing this page, we'd like to see you could keep it updated.

If you can do this shortly, you can just push it to this branch, and edit the PR description. Otherwise, you can open a new PR.

**Comment by yeya24** [comment]
@idealhack  Hi, I have updated the old version. If there is some problems, please check it out and tell me.
Thanks!

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 1]
```diff
@@ -8,172 +8,148 @@ content_template: templates/task
 {{% capture overview %}}
 
 
-Kubernetes 提供对分布在节点上的 NVIDIA GPU 进行管理的**实验**支持。本页描述用户如何使用 GPU 以及当前使用的一些限制
-
-{{% /capture %}}
-
-{{% capture prerequisites %}}
+Kubernetes 提供对分布在节点上的 AMD 和 NVIDIA GPU 进行管理的**实验**支持。在Kubernetes的v1.6版本中，加入了对NVIDIA GPU的支持，并且经历了多次向后的不兼容迭代。 而在v1.9版本中， 对AMD GPU的支持则通过 [device plugin](#deploying-amd-gpu-device-plugin) 的方式加入。
```
Comment (idealhack) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]: 对 NVIDIA GPU 的支持在 v1.6 中加入，已经经历了多次不向后兼容的迭代。而对 AMD GPU 的支持则在 v1.9 中通过 [device plugin](#deploying-amd-gpu-device-plugin) 加入。

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 2]
```diff
@@ -8,172 +8,148 @@ content_template: templates/task
...
 
+这个页面介绍了用户如何在不同的Kubernetes版本中使用 GPU， 以及当前存在的一些限制。
```
Comment (idealhack) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]: Please note the use of spaces: `这个页面介绍了用户如何在不同的 Kubernetes 版本中使用 GPU，以及当前存在的一些限制。`

**Comment by yeya24** [comment]
> I'm not sure if we have some guidelines about spaces here, but I think [this](https://github.com/sparanoid/chinese-copywriting-guidelines) is a good one, @yeya24 could you update accordingly?

Already update it. Please review it~

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 3]
```diff
@@ -8,172 +8,146 @@ content_template: templates/task
...
+# 安装 device plugin:
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/release-1.9/cluster/addons/device-plugins/nvidia-gpu/daemonset.yaml
+```
 
+在这一页面来提交相关 device plugin 的 issues 以及相应的安装方法 [GoogleCloudPlatform/container-engine-accelerators](https://github.com/GoogleCloudPlatform/container-engine-accelerators).
 
-到目前为止，还需要预先在节点上安装 CUDA 库
+## 集群内存在不同类型的 NVIDIA GPU
 
+如果集群内部的不同节点上有不同类型的 NVIDIA GPU，那么你可以使用 [Node Label 和 NodeSelecter](/docs/tasks/configure-pod-container/assign-pods-nodes/) 来将pod调度到合适的节点上。
```
Comment (idealhack) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]: `[Node Label 和 Node Selecter](/docs/tasks/configure-pod-container/assign-pods-nodes/)`

**Comment by yeya24** [comment]
thank u so much for your patience! @idealhack

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 4]
```diff
@@ -8,172 +8,146 @@ content_template: templates/task
...
 
+**从 1.8 版本往后，我们推荐通过 [device plugins](/docs/concepts/cluster-administration/device-plugins) 的方式来使用 GPU。**
 
-上述预备工作完成后，节点会自动发现它上面的 NVIDIA GPU，并将其作为可调度资源暴露
+在 1.10 版本之前，为了通过 device plugins 开启 GPU 的支持，我们需要在系统中将 `DevicePlugins`
+这一 feature gate 显式的设置为 true：`--feature-gates="DevicePlugins=true"`。不过，从 1.10 版本开始，我们就不需要这一步骤了。
 
-{{% /capture %}}
+接着你需要在主机节点上安装对应厂商的 GPU 驱动 并运行对应厂商的 device plugin
+([AMD](#deploying-amd-gpu-device-plugin), [NVIDIA](#deploying-nvidia-gpu-device-plugin))。
```
[the remark this answers is not in the document]
Comment (idealhack) [reply content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]: the links does not work yet

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 5]
```diff
@@ -8,172 +8,146 @@ content_template: templates/task
...
-```
+[官方的 NVIDIA GPU device plugin](https://github.com/NVIDIA/k8s-device-plugin) 有以下要求:
 
+- Kubernetes 的节点必须预先安装了 NVIDIA 驱动
+- Kubernetes 的节点必须预先安装 [nvidia-docker 2.0](https://github.com/NVIDIA/nvidia-docker)
+- 对于 docker 而不是 runc，nvidia-runtime 必须被设置为 [default-container-runtime](https://github.com/NVIDIA/k8s-device-plugin#preparing-your-gpu-nodes)
+- NVIDIA 驱动版本 ~= 361.93
 
-在 pod 上，通过节点[亲和性](/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity)规则为它指定可以使用的 GPU 类型
+要在你的集群已经启动的情况下部署 NVIDIA device plugin，可以运行下面的命令来实现:
```
[the remark this answers is not in the document]
Comment (idealhack) [reply content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]: sorry but i don't think this is resovled

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 6]
```diff
@@ -8,172 +8,144 @@ content_template: templates/task
...
 
+# 在 Ubuntu 上安装 NVIDIA 驱动 (实验性质):
+kubectl create -f https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/k8s-1.9/nvidia-driver-installer/ubuntu/daemonset.yaml
 
-## 访问 CUDA 库
+# 安装 device plugin:
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/release-1.9/cluster/addons/device-plugins/nvidia-gpu/daemonset.yaml
+```
 
+在这一页面来提交相关 device plugin 的 issues 以及相应的安装方法 [GoogleCloudPlatform/container-engine-accelerators](https://github.com/GoogleCloudPlatform/container-engine-accelerators)。
```
Comment (idealhack) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]: 请到 [GoogleCloudPlatform/container-engine-accelerators](https://github.com/GoogleCloudPlatform/container-engine-accelerators) 报告有关此 device plugin 以及安装方法的问题

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (idealhack) [reply content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]: @yeya24 I think it's the last one

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 7]
```diff
@@ -8,172 +8,144 @@ content_template: templates/task
 {{% capture overview %}}
 
 
-Kubernetes 提供对分布在节点上的 NVIDIA GPU 进行管理的**实验**支持。本页描述用户如何使用 GPU 以及当前使用的一些限制
-
-{{% /capture %}}
-
-{{% capture prerequisites %}}
+Kubernetes 提供对分布在节点上的 AMD 和 NVIDIA GPU 进行管理的**实验**支持。对 NVIDIA GPU 的支持在 v1.6 中加入，已经经历了多次不向后兼容的迭代。而对 AMD GPU 的支持则在 v1.9 中通过 [device plugin](#deploying-amd-gpu-device-plugin) 加入。
```
Comment (markthink) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]: 原始的英文怎么移掉了

Comment (idealhack) [reply content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]: 什么意思？

Comment (markthink) [reply content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]: 参考：https://github.com/k8smeetup/k8s-official-translation

Comment (idealhack) [reply content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]: 看起来这篇中文本来就没有带英文，而且相当老。

附带英文的做法在一开始可行，但会造成需要维护两份英文文章，可以改为加链接的方式，review 的时候点过去看对应英文就好。

**Comment by markthink** [comment]
@yeya24 不建议向 master 分支提交，目前社区讨论的结果是以 release-1.12 为基准进行更新，建议重新提交一个符合标准的 PR 至 release-1.12 分支。

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md (hunk 7 above)
Comment (markthink) [reply content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]: 之前的基准是 master 分支，往 master 分支提交确实会有这个问题，不过现在不是以 master 分支为基准..所以不存在此总是。还是按规范走，方便 reviewer 的人比对原文。

由于注释格式是一致的，脱敏移除英文也是非常容易的，退一看，不移除注释，浏览器渲染也不会把注释原文展示出来。

**Comment by idealhack** [comment]
我修改了分支， @yeya24 可以在 `release-1.12` HEAD 上 rebase

**Comment by idealhack** [comment]
@markthink 目前翻译的 PR 还有很多都指向 `master`。我们应该在 `website` 仓库或者 `community` 仓库中添加文档，说明翻译应该使用的分支规范，并且在 PR 的描述中提示 PR 作者。

**Comment by idealhack** [comment]
@yeya24 如有问题欢迎在 slack 或 wechat 联系我，如果放弃这个 PR，我可以新开一个 PR 将你作为 co-author

**Comment by idealhack** [comment]
Closing since https://github.com/kubernetes/website/pull/11182 has been merged.

/close

**Pull request closed without merging**